go-controller/pkg/ovn/pods.go

```diff
@@ -313,10 +313,13 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) (err error) {
 		return nil
 	}
 
+	var libovsdbExecuteTime time.Duration
+	var podAnnoTime time.Duration
 	// Keep track of how long syncs take.
 	start := time.Now()
 	defer func() {
-		klog.Infof("[%s/%s] addLogicalPort took %v", pod.Namespace, pod.Name, time.Since(start))
+		klog.Infof("[%s/%s] addLogicalPort took %v, libovsdbExecuteTime %v, pod Annotation time: %v",
+			pod.Namespace, pod.Name, time.Since(start), libovsdbExecuteTime, podAnnoTime)
```

Comment:
@dcbw : am I missing something? I remember you had added more verbose logs in one of your scale image test runs...wondering if there is something out of that worth capturing here by default?

Reply:
I think these are the two key ones. Everything else is tiny compared to these two.

But how about:

libovsdbExecuteTime -> libovsdb time
pod Annotation time -> annotation time

?

Reply:
ack let me fix the variable names, I like your suggestions better than the current ones.

Reply:
oh I think you meant the logs not variables, anyways will change both :D